cpp/memilio/mobility/graph.h

```diff
@@ -327,12 +327,14 @@ set_nodes(const Parameters& params, Date start_date, Date end_date, const fs::pa
  * @param[in] migrating_compartments Compartments that commute.
  * @param[in] contact_locations_size Number of contact locations.
  * @param[in] read_func Function that reads commuting matrices.
+ * @param[in] min_commuter_age minimum commuter age
+ * @param[in] max_comuter_age maximum commuter age
```

Comment:
We should maybe add an input vector of weights for commuting of the age groups. In the current implementation min, ..., max-1 commute fully while max commutes only with 0.33 weight. This is still hardcoded to AgeGroup 60-79.